config/tables.R

```diff
@@ -431,6 +431,65 @@ tables_config <- list(
     Shooting_count_actions = list(
         app_name = "Shooting Actions"
     ),
+
```

Comment:
Not a huge issue, but some of the spacing here is weird. Maybe we should look into a linter for R

Reply:
For this config file specifically, we should probably instead convert it to YAML or something a little more human-readable.

Reply:
I wouldn't mind YAML or TOML